other/materials_designer/create_interface_with_min_strain_zsl.ipynb

```diff
@@ -65,7 +65,7 @@
     "## 2. Set Interface Parameters\n",
```

Comment:
<p>This notebook needs updating</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>